charts/incubator/cloudflared/ci/test-values.yaml

```diff
@@ -1,3 +1,9 @@
-# With the bellow we test both container starts and that tpl is working inside extraArgs list
-args: ["tunnel", "--no-autoupdate", "run", "{{ .Values.testTpl }}"]
-testTpl: "--hello-world"
+workload:
+  main:
+    podSpec:
+      containers:
+        main:
+          args:
+            - tunnel
+            - --no-autoupdate
+            - --hello-world
```

Comment:
Let’s add a tpl “{{ }}” just to make sure it works (as it would on a normal deployment)

Reply:
I was thinking about that. There are multiple apps that use variables in args. Would it be better to add a test to common?

Reply:
Example: https://github.com/truecharts/charts/blob/master/charts/stable/adguard-home/values.yaml#L38

Reply:
It does have tests in common. (A lot more that a single string)
It’s just an extra line of defense

Reply:
Ok, give me a minute

Reply:
Done!. Tests still pass and CI still starts the tunnel.